lib/hubstats/github_api.rb

```diff
@@ -98,7 +98,8 @@ def self.update_teams
       grab_size = 250
       begin
         client = Hubstats::GithubAPI.client
```

Comment:
For some reason it wouldn't work correctly when we just did `client.organization_teams....`, but when we call the whole thing `Hubstats::GithubAPI.client.organization_teams....` it works. But I left the client variable since we still use it later on in the method.